docs/user_guide.rst

```diff
@@ -48,3 +48,4 @@ the RegionProcessor and validated using DataStructureDefinition.
   user_guide/model-mapping
   user_guide/model-registration
   user_guide/local-usage
+  user_guide/config
```

Comment:
I suggest to have "local-usage" last in this list.

Reply:
done